HEADER

```diff
@@ -1,6 +1,6 @@
 // This file is part of Substrate.
```

Comment:
It seems like this file isn't used anywhere. Can we rename it to `HEADER-APACHE` and also create a `HEADER-GPL`?

Reply:
I don't understand this file either. We could also remove it 🤷

Reply:
It is for benchmarks template

Reply:
@shawntabrizi isn't that `frame-weight-template.hbs`?

Reply:
Ah yeah maybe I don't use it anymore :) None the less, isnt it good to have? Polkadot has one too

Reply:
The point is that the header is different between the client/runtime.

I'm fine with keeping it for now.

Reply:
I assumed that this existed for people to copy/paste into files. That's why I suggested creating both files, like we have `LICENSE-GPL3` and `LICENSE-APACHE2`.

Reply:
That sounds super reasonable to me

Reply:
Done.